src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/SetAzureRmSqlDatabaseBackupShortTermRetentionPolicy.cs

```diff
@@ -0,0 +1,110 @@
+        SupportsShouldProcess = true),
+    OutputType(typeof(AzureSqlDatabaseBackupShortTermRetentionPolicyModel))]
+    public class SetAzureRmSqlDatabaseBackupShortTermRetentionPolicy : AzureSqlDatabaseBackupShortTermRetentionPolicyCmdletBase
+    {
+        /// <summary>
+        /// Gets or sets backup retention days.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            Position = 3,
+            HelpMessage = "The backup retention setting, in days.")]
+        [ValidateNotNullOrEmpty]
+        [ValidateRetentionDays]
+        public int RetentionDays{ get; set; }
+
+        /// <summary>
+        /// Get the entities from the service
+        /// </summary>
+        /// <returns>The list of entities</returns>
+        protected override IEnumerable<AzureSqlDatabaseBackupShortTermRetentionPolicyModel> GetEntity()
+        {
+            if (InputObject != null)
+            {
+                ResourceGroupName = InputObject.ResourceGroupName;
+                ServerName = InputObject.ServerName;
+                DatabaseName = InputObject.DatabaseName;
+            }
+
+            ICollection<AzureSqlDatabaseBackupShortTermRetentionPolicyModel> results = new List<AzureSqlDatabaseBackupShortTermRetentionPolicyModel>()
+            {
+                ModelAdapter.GetDatabaseBackupShortTermRetentionPolicy(
+                    this.ResourceGroupName,
+                    this.ServerName,
+                    this.DatabaseName)
+            };
+
+            return results;
+        }
+
+        /// <summary>
+        /// Create the model from user input
+        /// </summary>
+        /// <param name="model">Model retrieved from service</param>
+        /// <returns>The model that was passed in</returns>
+        protected override IEnumerable<AzureSqlDatabaseBackupShortTermRetentionPolicyModel> ApplyUserInputToModel(IEnumerable<AzureSqlDatabaseBackupShortTermRetentionPolicyModel> model)
+        {
+            return new List<AzureSqlDatabaseBackupShortTermRetentionPolicyModel>()
+            {
+                new AzureSqlDatabaseBackupShortTermRetentionPolicyModel(
+                    ResourceGroupName,
+                    ServerName,
+                    DatabaseName,
+                    new Management.Sql.Models.BackupShortTermRetentionPolicy(retentionDays: RetentionDays))
+            };
+        }
+
+        /// <summary>
+        /// Update the entity
+        /// </summary>
+        /// <param name="entity">The output of apply user input to model</param>
+        /// <returns>The input entity</returns>
+        protected override IEnumerable<AzureSqlDatabaseBackupShortTermRetentionPolicyModel> PersistChanges(IEnumerable<AzureSqlDatabaseBackupShortTermRetentionPolicyModel> entity)
+        {
+            if (ShouldProcess(DatabaseName))
+            {
+                return new List<AzureSqlDatabaseBackupShortTermRetentionPolicyModel>() {
```

Comment:
May be Invert 'if' statement to reduce nesting?
```Cs
if (!ShouldProcess(DatabaseName)) return null;
```

Reply:
Agreed.